norsk-henvisningsstandard-for-rettsvitenskapelige-tekster.csl

```diff
@@ -0,0 +1,318 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="nb-NO" demote-non-dropping-particle="display-and-sort" page-range-format="expanded">
+  <info>
+    <title>Norsk henvisningsstandard for rettsvitenskapelige tekster (Norsk - Bokmål)</title>
+    <title-short>Norsk rettsvitenskap</title-short>
+    <id>http://www.zotero.org/styles/norsk-henvisningsstandard-for-rettsvitenskapelige-tekster</id>
+    <link href="http://www.zotero.org/styles/norsk-henvisningsstandard-for-rettsvitenskapelige-tekster" rel="self"/>
+    <link href="https://www.uio.no/for-ansatte/enhetssider/jus/ledelses-og-utvalgsmoter/pmr/moter/2018/pmr5/vedlegg/sak9.pdf" rel="documentation"/>
+    <author>
```

Comment:
Could you move Hans Gunnar into a contributor role? It's helpful if the author is the maintainer of the style (since we don't have separate roles for maintainers)

Reply:
Done.